test/typescript/actionCreators.ts

```diff
@@ -4,7 +4,7 @@ import {
   Dispatch,
   bindActionCreators,
   ActionCreatorsMapObject
-} from 'redux'
+} from '../..'
```

Comment:
Thanks. This could have caused a number of problems for people working on the code!